test/datalog_tests/simple2.dl

```diff
@@ -42,3 +45,18 @@ output relation TArrng1Arrng2(x: bigint)
 TArrng1Arrng2(x.f2.f2) :-
     TArrng1[t],
     TArrng2[(x, t.0.f2.f2)].
+
+
+function ftoi_(l: double): signed<32> = {
```

Comment:
you could also uncomment this in the souffle_lib.dl file

Reply:
Do we have/need tests for that?

Reply:
Previously it wouldn't even compile (the generated Rust). So that's one test.
Second, if that works I may be able to enable at least one Souffle test that is using it (if everything else works too).

Reply:
Ok, uncommented `ftoi`, `ftou`. Will merge once tests pass.